### PR TITLE
Define import property

### LINKED ISSUE #74: HTML Import feature detection does not detect this polyfill
The recommended way for checking for html import support is by creating a link element and checking for the 'import' property (i.e. used by webcomponents-loader.js https://github.com/webcomponents/webcomponentsjs/blob/35acb3135c18b88bb46a737f406cbfabe5eac05c/webcomponents-loader.js#L20). However, when the html import polyfill is loaded, it never defines the import property, meaning that the loader script will fail to detect html imports, which are not supported (if something causes the loader script to run again; in my case this happens during the import of nested components). You could argue that the main problem is the duplicate include of the loader, but I don't see why the webcomponents polyfill should not correctly detect html imports provided by itself. Custom elements and shadow dom are correctly detected.

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Fixes #74 
Define the `import` read-only property on `HTMLLinkElement` instances.

### CODE REVIEW

**Review by TimvdLippe — APPROVED** [review]
Nice! @azakus could you quickly check as well?

File: gulpfile.js [hunk 1]
```diff
@@ -10,26 +10,31 @@
...
+  compilation_level: 'ADVANCED',
+  warning_level: 'VERBOSE',
+  language_in: 'ES6_STRICT',
+  language_out: 'ES5_STRICT',
+  externs: ['externs/html-imports.js'],
+  js_output_file: 'html-imports.min.js',
+  assume_function_wrapper: true,
+  new_type_inf: true,
+  rewrite_polyfills: false,
+  new_type_inf: true,
```
Comment (dfreedm) [inline gulpfile.js]: `new_type_inf` is in here twice.

File: gulpfile.js [hunk 2]
```diff
@@ -10,26 +10,31 @@
...
+  module_resolution: 'NODE',
```
Comment (dfreedm) [inline gulpfile.js]: This line is unnecessary I think? DIdn't see any imports in the main file

**Review by dfreedm — CHANGES_REQUESTED** [review]

File: gulpfile.js (hunk 2 above)
Comment (valdrinkoshi) [reply gulpfile.js]: The copy-paste curse :x will fix

**Review by dfreedm — APPROVED** [review]

File: gulpfile.js (hunk 2 above)
Comment (valdrinkoshi) [reply gulpfile.js]: Done in https://github.com/webcomponents/html-imports/pull/75/commits/b14476ce0dc5dd4b1d93469d7abf79cd1f5a4f01, PTAL

**Pull request merged**